hypha/apply/dashboard/templates/dashboard/applicant_dashboard.html

```diff
@@ -34,8 +34,8 @@ <h2 class="font-light flex-1">{% trans "My active submissions" %}</h2>
                         {% for submission in my_active_submissions %}
                             <div class="wrapper wrapper--status-bar-outer">
                                 <div class="wrapper wrapper--status-bar-inner">
-                                    <div class="mt-5 ml-4">
-                                        <h4 class="heading mb-2 font-bold"><a class="link link--underlined" href="{% url 'funds:submissions:detail' submission.id %}">{{ submission.title }}</a></h4>
+                                    <div class="mt-5 ml-4 lg:max-w-[30%]">
```

Comment:
```suggestion
                                    <div class="mt-4 ml-4 lg:max-w-1/3 lg:mt-0">
```
Update to remove margin top to give it some more room and use more standard 1/3 tailwind class.

Reply:
Top margin also helps to align it with the status bar, especially on large screens.